src/todo/todo.lua

```diff
@@ -134,8 +134,12 @@ function todo.import_tasks(dialog)
     todo.log(tasks)
 
     for _, task_to_import in pairs(tasks) do
-        local task = todo.create_task(task_to_import.task)
-        -- TODO: update modified by/created_by
+        local task = todo.create_task(task_to_import.task, "", player)
```

Comment:
You can pass in nil here to save the assignee line below.

Reply:
Done.